createlinks

```diff
@@ -24,6 +24,7 @@ use esmith::Build::CreateLinks  qw(:all);
 
 my @templates = (qw(
                 /etc/hosts
+                /etc/dnsmasq.conf
```

Comment:
If we expand this template we invert the dependency between nethserver-hosts and nethserver-dnsmasq RPMs. As nethserver-dnsmasq is today an ISO package we can do it. Could you analyse the consequences of inverting the dependency direction?

Reply:
Ok just tested it is good for me

If you add `Requires: nethserver-dnsmasq` to the spec file of nethserver-hosts you have an error, therefore you must also remove the `Requires: nethserver-hosts` of the nethserver-dnsmasq.spec

Can we go for this ?

Reply:
after tested it, I'm sure it worth the game to inverse the dependency
Actually (nethserver-dnsmasq requires nethserver-hosts)
if I remove nethserver-dnsmasq, only nethserver-dnsmasq is removed
if I remove nethserver-hosts, both nethserver-hosts and nethserver-dnsmasq are removed

if I inverse the dependency (nethserver-hosts requires nethserver-dnsmasq)
if I remove nethserver-dnsmasq, both nethserver-hosts and nethserver-dnsmasq are removed
if I remove nethserver-hosts, both nethserver-hosts and nethserver-dnsmasq are removed